files/acl/restrict-unpublished-files.php

```diff
@@ -13,6 +13,10 @@
 const CACHE_GROUP = 'vip-files-acl';
 
 function check_file_visibility( $file_visibility, $file_path ) {
+	if ( 0 === strpos( $file_path, 'sites/' ) ) {
+		$file_path = preg_replace( '#^sites\/\d+\/#', '', $file_path );
```

Comment:
Redundant forward-slash escaping. Additionally, I think some comments are in order as well as PHPDoc for the function and type hints in the function signature.

Reply:
```suggestion
		$file_path = preg_replace( '#^sites/\d+/#', '', $file_path );
```